### ISSUE TITLE
Get-ATPProtectionPolicyRule fails when running Get-ORCAReport

### ISSUE THREAD

**Issue by boyedarat** [cammurray/orca#166]
When running Get-ORCAReport, the following output is produced in the console:
```
...
01/05/2023 12:01:59 Getting Anti-Spam Settings
01/05/2023 12:02:00 Getting Tenant Settings
01/05/2023 12:02:00 Getting ATP Preset Policy Settings
Get-ATPProtectionPolicyRule: The term 'Get-ATPProtectionPolicyRule' is not recognized as a name of a cmdlet, function, script file, or executable
program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
01/05/2023 12:02:00 Getting EOP Preset Policy Settings
...

```
Then viewing the HTML report, we get the following warning:
![image](https://user-images.githubusercontent.com/22816484/210701159-af68052d-6a49-4662-80e1-947ae814a3c7.png)

We do have Defender for Office 365 enabled on our tenant and the View-Only permission assigned to the ID. So this seems like some permissions or role is missing from the ID we are using to run the scan. But the console error (as shown in the output above) gives us pause that it may be an issue with our PS setup instead.

Could you advise us on how to root cause the issue (e.g.: permissions issue or PS issue) and how to resolve it?

Thanks!

**Comment by cammurray** [comment]
Hi @boyedarat the Get-ATPProtectionPolicyRule error in your case will be fixed with #179 - we run that command at the moment even when we don't detect ATP.

On your second issue here, which is I assume is your main concern, is that ORCA isn't detecting MDO in your environment. We look to see if you can run the Get-AtpPolicyForO365 command, which will only be present if you a) have licenses for MDO and b) have permissions to run that command. I suspect you don't have permissions to run that command, do you only have view-only role?

**Comment by cammurray** [comment]
Closing off as no response from author on the permission issue, feel free to re-open if still an issue.
